independent-projects/resteasy-reactive/common/runtime/src/main/java/org/jboss/resteasy/reactive/common/util/RestMediaType.java

```diff
@@ -0,0 +1,18 @@
+package org.jboss.resteasy.reactive.common.util;
+
+import javax.ws.rs.core.MediaType;
+
+/**
+ * Extended media types in Resteasy Reactive.
+ */
+public final class RestMediaType {
```

Comment:
Any reason why this is not a subtype of `MediaType`?

Reply:
Sorry, I've missed this change. PR updated!